core/aggregator/aggregators.go

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) 2008-2018, Hazelcast, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License")
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package aggregator
+
+import (
+	"github.com/hazelcast/hazelcast-go-client/internal/aggregation"
+)
+
+// Count returns a Count aggregator with the given attribute path.
+func Count(attributePath string) interface{} {
```

Comment:
Why do they return `interface{}`? It should return `*aggregation.Count` and same for the followings.